composer.json

```diff
@@ -98,7 +101,8 @@
       "!.wordpress-org/",
       "docs/",
       "wp-graphql.zip",
-      "!build"
+      "!build",
+      "packages"
```

Comment:
this excludes the `packages` directory when the plugin is archived, to be ready to be deployed to WordPress.org

The directory is no longer ignored when the plugin is cloned/downloaded, so the JS app can be built, but we don't want to include the raw JS in the distributed plugin.